drivers: led: fix LED_BRIGTHNESS_MAX typo

kartben opened the pull request:
Renames LED_BRIGTHNESS_MAX to LED_BRIGHTNESS_MAX in led.h and all in-tree users of the macro.

faxe1008 reviewed (dismissed):
I think the migration guide is not needed since the macro was introduced this release cycle?

kartben commented:
> I think the migration guide is not needed since the macro was introduced this release cycle?

Oh was it? Didn't really thought of checking. Then I'll also drop the deprecation and go for a simple rename :)

kartben commented:
> I think the migration guide is not needed since the macro was introduced this release cycle?

Oh was it? Didn't really thought of checking. Then I'll also drop the deprecation and go for a simple rename :)

kartben committed:
drivers: led: fix LED_BRIGTHNESS_MAX typo

Renames LED_BRIGTHNESS_MAX to LED_BRIGHTNESS_MAX in led.h and all
in-tree users of the macro.

Signed-off-by: Benjamin Cabé <benjamin@zephyrproject.org>

fabiobaltieri reviewed (approved):
palmface

simonguinot reviewed (dismissed):
Thanks !
